inc/namespace.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+namespace Altis\Security\Browser;
+
+use Altis;
+use WP_Dependencies;
+use WP_Error;
+
+const INTEGRITY_DATA_KEY = 'altis_integrity_hash';
+const INTEGRITY_HASH_ALGO = 'sha384';
```

Comment:
Any reason why this one specifically? Splitting the difference between best and fastest?

Reply:
This seems to be the most commonly used throughout all the examples and existing code out there, so stuck to it.

Reply:
Updated the workflows one to match